src/global.d.ts

```diff
@@ -0,0 +1 @@
+type Nullable<T> = T | null | undefined;
```

Comment:
Looks to me like this is a custom type that can be used anywhere in the code base (for example, [here](https://github.com/microbiomedata/nmdc-field-notes/pull/18/files#diff-769911c416ccf8514d8fd941ae0abe8fb5c606ade0c218e22151a5f5f9f3d700R18)) without having to `import` it there.